### DISCUSSION TITLE
v1 post build commands to move folder?

### DISCUSSION THREAD

**Discussion by ssuess** [quasarframework/quasar#13678] [CLI - PWA mode] [answered]
Hi, there is one folder in my `/public` directory that gets included in my PWA builds inside the particular build folder, but because of the way I have this setup on my server, I would like it to live outside that build folder. Is there some post build command (or build config) that I can use to move this automatically at the end of the build? Right now I am moving it by hand.

**Comment by MilosPaunovic** [comment, marked as answer]
You could make use of `afterBuild` hook.
[https://v1.quasar.dev/quasar-cli/quasar-conf-js#property-build](https://v1.quasar.dev/quasar-cli/quasar-conf-js#property-build)
